shared/src/commonMain/kotlin/org/hyperskill/app/step_quiz_code_blanks/presentation/StepQuizCodeBlanksFeature.kt

```diff
@@ -37,6 +37,9 @@ object StepQuizCodeBlanksFeature {
             internal val codeBlanksVariablesSuggestions: List<Suggestion.ConstantString> =
                 step.codeBlanksVariablesSuggestions()
 
+            internal val codeBlanksVariablesAndStringsSuggestions: List<Suggestion.ConstantString> =
+                codeBlanksVariablesSuggestions + codeBlanksStringsSuggestions
+
```

Comment:
Can we extract all these fields into extensions?
It is a common practice in kotlin to not store logic in the data class.

Reply:
An extension property cannot have stored properties. If we move this to an extension, it will be recomputed each time.